tests/test_rum_injection.py

```diff
@@ -0,0 +1,43 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+"""Tests for RUM (Real User Monitoring) support scenario"""
+
+from html.parser import HTMLParser
+
+from utils import weblog, scenarios, features
+
+
+@scenarios.rum_injection
+@features.not_reported
```

Comment:
You should register the new feature in the Feature Parity DashBoard
https://github.com/DataDog/system-tests/blob/main/docs/edit/features.md